### PR TITLE
Add support for job_cmd job data argument and non-ascii values

### PR DESCRIPTION
**Pull request by chekunkov**
If it looks good please release new version of the image and update https://github.com/scrapinghub/custom-images-examples/pull/8

### CODE REVIEW

File: bin/env2options [hunk 1]
```diff
@@ -78,17 +89,20 @@ def parse_args():
 def main():
     args = parse_args()
     if args.source == SOURCE_ALL:
-        values = get_merged_settings()
+        job_cmd = get_job_cmd()
+        merged_settings = get_merged_settings()
```
Comment (vshlapakov) [inline bin/env2options:93]: @dangra I've just noticed that shub settings are merged [incorectly](https://github.com/scrapinghub/scrapinghub-image-casperjs/blob/master/bin/env2options#L25-L33) - it assumes there's a dict with string keys and dict values and merges it one by one in alphabetical order. 

While in real, 1) there should be strict order of merging, cause `job_settings` will be overwritten with `project settings`, etc 2) not all values are dicts - I guess we agreed to drop addons and `addons_settings` for non-Scrapy, but there're some others. 

I wonder if there was a discussion about it already, or there're some agreed assumptions I'm missing? We probably should hard-code settings section names somewhere and sort it properly.

**Review by vshlapakov — APPROVED** [review]
I'm inclined to merge this PR tomorrow ~~as-is and create another one to fix the settings issue if needed~~, let me know if there's any pushback.

File: bin/env2options (hunk 1 above)
Comment (dangra) [reply bin/env2options:93]: I am not sure we are reading the same code for get_merged_settings.

My understanding is that settings are merged in the order defined by https://github.com/scrapinghub/scrapinghub-image-casperjs/blob/3dfa56db3eebeebfc1397b242caf6d80b99dc264/bin/env2options#L17-L22

Comment (vshlapakov) [reply bin/env2options:93]: Ohh, I don't know how it happened, sorry for this noise :man_facepalming:

**Pull request merged**